src/ray/rpc/grpc_client.h

```diff
@@ -88,6 +88,51 @@ inline std::shared_ptr<grpc::Channel> BuildChannel(
   return channel;
 }
 
+inline grpc::ChannelArguments CreateClientDefaultChannelArguments() {
+  // Please refer to explanation of gRPC returned statuses for more details:
+  // REF: https://grpc.github.io/grpc/core/md_doc_statuscodes.html
+  std::string service_config_json = R"(
+  {
+    "methodConfig": [{
+      "name": [
+        {"service": "ray.rpc.CoreWorkerService", "method": "ReportGeneratorItemReturns"},
```

Comment:
I'm actually not sure that ReportGeneratorItemReturns is idempotent for the same task execution. @rkooo567 ?